test/utils.ts

```diff
@@ -0,0 +1,30 @@
+import { observable, computed, Computed } from "knockout";
+
+// delays the execution of f a number of times.
+export function execAt(f, stop) {
```

Comment:
I see this function is new, but not being called anywhere (same applies for the other function in this utils file. Is it me or they should not be included in the codebase yet?

Reply:
ok